api/app/src/command/medical/document/document-redownload.ts

```diff
@@ -0,0 +1,286 @@
+import {
+  Coding,
+  DocumentReference,
+  DocumentReferenceContent,
+  Identifier,
+  Resource,
+} from "@medplum/fhirtypes";
+import {
+  Contained,
+  Document,
+  DocumentContent,
+  DocumentIdentifier,
+} from "@metriport/commonwell-sdk";
+import { difference, differenceBy, groupBy } from "lodash";
+import { DeepRequired } from "ts-essentials";
+import { downloadDocsAndUpsertFHIR } from "../../../external/commonwell/document/document-query";
+import { hasCommonwellContent, isCommonwellContent } from "../../../external/commonwell/extension";
+import { makeFhirApi } from "../../../external/fhir/api/api-factory";
+import { downloadedFromHIEs } from "../../../external/fhir/shared";
+import { isMetriportContent } from "../../../external/fhir/shared/extension";
+import { decodeExternalId } from "../../../shared/external";
+import { capture } from "../../../shared/notifications";
+import { Util } from "../../../shared/util";
+import { getOrganizationOrFail } from "../organization/get-organization";
+import { getPatientOrFail } from "../patient/get-patient";
+import { updateDocQuery } from "./document-query";
+
+export const reprocessDocuments = async ({
```

Comment:
Question, I dont see this being used anywhere are we planning on adding this somewhere?

Reply:
Its being called from the internal route `POST /docs/reprocess`